client/components/plans/plan-price/style.scss

```diff
@@ -1,32 +1,46 @@
-.plan-header .plan-price {
+.plan-header .plan-price,
```

Comment:
We spoke about this and it seems like this approach is the simplest one for now. Trying to avoid premature optimization :)